src/fairseq2/models/wav2vec2/factory.py

```diff
@@ -222,6 +222,32 @@ def _base() -> Wav2Vec2Config:
     return Wav2Vec2Config()
 
 
+@wav2vec2_arch("large_ls960")
+def _large_ls960() -> Wav2Vec2Config:
```

Comment:
Nit: Since the meaning of "ls" and "lv" prefixes are not immediately clear, could you please add a short doctstr to these two config functions, referring to which setting they refer in the original paper?